### ISSUE TITLE
Terminology improvements

### ISSUE THREAD

**Issue by guardrex** [dotnet/aspnetcore#23711]
We've cleaned up "whitelist"/"blacklist" language in the docs (the Core docs anyway) on https://github.com/dotnet/AspNetCore.Docs/pull/19115. I isolated 15 instances of these terms in the framework. Recommend where possible that these be changed to eliminate the words ...

* https://github.com/dotnet/aspnetcore/search?q=whitelist&unscoped_q=whitelist
* https://github.com/dotnet/aspnetcore/search?q=blacklist&unscoped_q=blacklist

Regarding "master-slave" terminology: Doc guidelines warn off of "slave" in most cases. Docs and the framework don't use the word "slave" anywhere outside of a couple of issue remarks.

**Comment by javiercn** [comment]
@guardrex thanks for bringing this up.

We've already looked into things like this, unfortunately the references you mentioned here all belong to external libraries (bootstrap, react, etc.)

We've already cleaned up all the sensitive language from our repo code, in fact we have tooling that runs and identifies problematic terms for us.

@Pilchie I don't think there's anything left we can do here, but I bring this up to your attention in case you have some other ideas.
